UnitTests/Drawing/AttributeTests.cs

```diff
@@ -47,7 +50,7 @@ public void ColorNamesConstructor ()
 		Assert.Equal (new Color (Color.Blue), attribute.Background);
 	}
 
-	[Fact, AutoInitShutdown]
+	[Fact] [AutoInitShutdown]
```

Comment:
If we do a format with Ctrl+K/Ctrl+D it will change to this:

```cs
[Fact]
[AutoInitShutdown]
```

I really don't understand why you can't see them as `[Fact, AutoInitShutdown]`.